_drafts/oryx/packages.md

```diff
@@ -0,0 +1,99 @@
+platform --> base
+domain --> base
+```
+
+### Template packages
+
+The template layer contains packages that can be used as quick starters for demos and projects. Templated packages do follow semantic versioning and will deliver on the promise of upgradability, but are fairly opinionated as they mainly serve to get up to speed fast with a standard frontend application.
+
+| Packages                                                       | Location                    |
+| -------------------------------------------------------------- | --------------------------- |
+|                                                                |                             |
+| [Application](https://www.npmjs.com/package/@spryker-oryx/)    | `@spryker-oryx/application` |
+| [Presets](https://www.npmjs.com/package/@spryker-oryx/presets) | `@spryker-oryx/presets`     |
+| [Labs ](https://www.npmjs.com/package/@spryker-oryx/labs)      | `@spryker-oryx/labs`        |
+| [Themes ](https://www.npmjs.com/package/@spryker-oryx/themes)  | `@spryker-oryx/themes`      |
+
+> **Note:** the labs package is an exception here. The labs package consists of experimental or demo functionality and is not recommended to be used in production.
+
+### Domain packages
+
+Domain packages provide components, services and adapters for a certain domain. Organising packages in domains helps the developer experience, as its' fairly easy to understand where to find a certain component or service. An example of a domain package is the `product` package, which contains all product related components as well as the product services and adapters that integrates with Spryker APIs.
+
+| Packages                                                         | Location                 |
+| ---------------------------------------------------------------- | ------------------------ |
+| [Auth](https://www.npmjs.com/package/@spryker-oryx/auth)         | `@spryker-oryx/auth`     |
+| [Cart](https://www.npmjs.com/package/@spryker-oryx/cart)         | `@spryker-oryx/cart`     |
+| [Checkout](https://www.npmjs.com/package/@spryker-oryx/checkout) | `@spryker-oryx/checkout` |
+| [Content](https://www.npmjs.com/package/@spryker-oryx/content)   | `@spryker-oryx/content`  |
+| [Order](https://www.npmjs.com/package/@spryker-oryx/order)       | `@spryker-oryx/order`    |
+| [picking](https://www.npmjs.com/package/@spryker-oryx/picking)   | `@spryker-oryx/picking`  |
+| [Product](https://www.npmjs.com/package/@spryker-oryx/product)   | `@spryker-oryx/product`  |
+| [Search](https://www.npmjs.com/package/@spryker-oryx/search)     | `@spryker-oryx/search`   |
+| [Site](https://www.npmjs.com/package/@spryker-oryx/site)         | `@spryker-oryx/site`     |
+| [User](https://www.npmjs.com/package/@spryker-oryx/user)         | `@spryker-oryx/user`     |
+
+### Platform packages
+
+The platform layer contains the core packages, including `core` itself. They are providing the infrastructure to the whole system.
```

Comment:
`The platform layer contains the core packages, including core itself` can be interpreted wrongly: we have core package and also core packages. Maybe we can rephrase?